unit-tests/syncer/sw.py

```diff
@@ -61,6 +64,46 @@ def init():
     #
     global syncer
     syncer = rs.syncer( 100 )  # We don't want to lose any frames so uses a big queue size (default is 1)
+    #
+    global playback_status
+    playback_status = None
+
+
+def playback_callback( status ):
+    """
```

Comment:
Why do we need 2 x """ on each function begin?

Reply:
Placeholder for comments which I was too lazy to fill in...